gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 javaVersion = "17"
 
-kotlinVersion = "1.8.22" # Kotlin, Kotlin Reflect and Dokka versions should have the same minor version to support the same kotlin version
+kotlinVersion = "1.9.25" # Kotlin, Kotlin Reflect and Dokka versions should have the same minor version to support the same kotlin version
```

Comment:
NIT. As far as I see, we don't use Kotlin Reflect now. If so, we can drop it from the comment.